src/routes/+layout.svelte

```diff
@@ -14,25 +15,5 @@
     <slot />
 </main>
 <footer class="p-8 sm:p-12 md:p-16">
-    <ul class="flex flex-row flex-wrap justify-center gap-8">
-        <!-- TODO: Use actual SVG social icons here. -->
-        <li><a href="mailto:lino@up-csi.org">lino@up-csi.org</a></li>
-        <li>
-            <a target="_blank" rel="noopener" href="https://www.facebook.com/upcsi">Facebook</a>
-        </li>
-        <li>
-            <a target="_blank" rel="noopener" href="https://twitter.com/officialupcsi">Twitter</a>
-        </li>
-        <li>
-            <a
-                target="_blank"
-                rel="noopener"
-                href="https://www.linkedin.com/company/up-center-for-student-innovations/"
-                >LinkedIn</a
-            >
-        </li>
-        <li>
-            <a target="_blank" rel="noopener" href="https://github.com/up-csi ">GitHub</a>
-        </li>
-    </ul>
+    <SidesFooter />
```

Comment:
We could actually just put the `<footer>` inside the `SidesFooter` component itself. Instead of having a `<div>` as the root, we can just use `<footer>` instead.